integrations/influxdb_connector/influxdb-timestream-connector/README.md

```diff
@@ -47,30 +47,70 @@ The InfluxDB Timestream connector can be deployed within an AWS CloudFormation s
 
-1. Run the following command replacing `<region>` with the AWS region you want to deploy in and provide parameter overrides as desired:
+##### Stack Parameters
+
+The following parameters are available when deploying the connector as part of a CloudFormation stack. An example of setting these parameters is included in step 4 of the [SAM deployment steps](#sam-deployment-steps).
+
+| Parameter     | Description | Default Value |
+|---------------|-------------|---------------|
+| `DatabaseName`  | The name of the database to use for ingestion. | `influxdb-line-protocol` |
+| `LambdaMemorySize` | The size of the memory in MB allocated per invocation of the function. | `512` |
+| `LambdaName` | The name to use for the Lambda function. | `influxdb-timestream-connector-lambda` |
+| `LambdaTimeoutInSeconds` | The number of seconds to run the Lambda function before timing out. | `30` |
+| `MeasureNameForMultiMeasureRecords` | The value to use in records as the `measure_name`, as shown in the [example line protocol to Timestream records translation](#resulting-cpu_load_short-timestream-for-liveanalytics-table). | `influxdb-measure` |
+| `RestApiGatewayName` | The name to use for the REST API Gateway. | `InfluxDB-Timestream-Connector-REST-API-Gateway` |
+| `RestApiGatewayStageName` | The name to use for the REST API Gateway stage. | `dev` |
+| `RestApiGatewayTimeoutInMillis` | The maximum number of milliseconds a REST API Gateway event will wait before timing out. | `30000` |
+| `WriteThrottlingBurstLimit` | The number of burst requests per second that the REST API Gateway permits. | `1200` |
+
+##### SAM Deployment Steps
+
+1. [Download and install the AWS SAM CLI](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/install-sam-cli.html).
+2. [Download and install Rust](https://www.rust-lang.org/tools/install).
+3. [Download and install Cargo Lambda](https://www.cargo-lambda.info/guide/installation.html).
+4. Run the following command to package the binary in `target/lambda/influxdb-timestream-connector/bootstrap.zip`, where `template.yml` expects it to be, and cross compile for Linux ARM:
+    ```
+    cargo lambda build --release --arm64 --output-format zip
+    ```
+5. Run the following command, replacing `<region>` with the AWS region you want to deploy in, `<stack name>` with your desired stack name and providing parameter overrides as desired. Note that this example command uses `--resolve-s3` to automatically create an S3 bucket to use for packaging and deployment and `--capabilities CAPABILITY_IAM` to allow the creation of IAM roles.
 
     ```shell
-    sam --region <region> --parameter-overrides ParameterKey=exampleKey,ParameterValue=exampleValue deploy template.yml
+    sam deploy template.yml \
```

Comment:
Add instruction on how to run tests

Reply:
I have added instructions in the README for how to run tests, including how to run all tests.